cypress/e2e/po/lists/repositories-list.po.ts

```diff
@@ -0,0 +1,7 @@
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+
+export default class RepositoriesListPo extends BaseResourceList {
```

Comment:
same naming convention for this one, suggestion was ChartRepositories

Reply:
@richard-cox looks like there was an existing po for this `cypress/e2e/po/lists/catalog.cattle.io.clusterrepo.po.ts` so I'll use that one instead of the one I created and rename it. `RepoListPo` to `ChartRepositoriesListPo`

Reply:
should i also update the fiile name from `catalog.cattle.io.clusterrepo.po` to `chart-repositories.po`?